### DISCUSSION TITLE
Entry_bar: Time of a Closed Trade

### DISCUSSION THREAD

**Discussion by umashgh** [kernc/backtesting.py#540] [Q&A] [answered]
Hi,

I am trying to access closed trades from _strategy object. The time is mentioned as entry_bar to exit_bar. How do I convert this to clock time?

Also related: Can I access the bar info in strategy.next() function? So as to backtest strategies that depend on the nth candle of the day?

**Comment by kernc** [comment, marked as answer]
> How do I convert this to clock time?

You can use entry_bar and exit_bar as indexes into `Strategy.data.index`.

> nth candle of the day

See https://github.com/kernc/backtesting.py/issues/188#issuecomment-739582424, https://github.com/kernc/backtesting.py/issues/502#issuecomment-939561481.
